components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIGatewayManager.java

```diff
@@ -212,12 +223,24 @@ public void removeFromGateway(API api, String tenantDomain) throws Exception {
                     client.deleteDefaultApi(tenantDomain);
                 }
             }
+                } catch (AxisFault axisFault) {
```

Comment:
Put comments to explain why we are logging and continuing instead of throwing the exception.